pkg/version/version.go

```diff
@@ -199,3 +224,20 @@ func CheckTiDBVersion(versionStr string, requiredMinVersion, requiredMaxVersion
 	}
 	return CheckVersion("TiDB", *version, requiredMinVersion, requiredMaxVersion)
 }
+
+// NormalizeBackupVersion normalizes the version string from backupmeta.
+func NormalizeBackupVersion(version string) *semver.Version {
```

Comment:
Seem `version.removeVAndHash` can be replaced by this funcation.

Reply:
You mean merge those two functions? Seems `removeVAndHash` focus on the version string made by the make script, and `NormalizeBackupVersion` focus on the version string returned from TiKV. Maybe it's hard to just replace the former with the latter. 🤔